doc/nrf/device_guides/working_with_nrf/nrf53/nrf5340.rst

```diff
@@ -579,6 +579,57 @@ Bluetooth buffers configuration introduced by the :kconfig:option:`CONFIG_NCS_SA
    :start-after: fota_upgrades_zigbee_start
    :end-before: fota_upgrades_zigbee_end
 
+.. _ug_nrf5340_image_slot_numbering:
+
+Image IDs and slot numbering
+****************************
+
+Each image in the DFU process on the nRF5340 DK is identified by a unique ID, corresponding to a specific memory slot for each core.
+The slots are numbered from 0, and each slot corresponds to a specific location in memory.
+This configuration typically follows:
+
++-------+-------------------------------------------------------------+-----------+-------+
+| Slot  | Description                                                 | DTS label | n     |
++=======+=============================================================+===========+=======+
+| slot0 | First image's primary slot (internal flash)                 | "image-0" | 1     |
++-------+-------------------------------------------------------------+-----------+-------+
+| slot1 | First image's secondary slot (internal or external flash)   | "image-1" | 0, 2  |
++-------+-------------------------------------------------------------+-----------+-------+
+| slot2 | Second image's primary slot (internal RAM)                  | "image-2" | 3     |
++-------+-------------------------------------------------------------+-----------+-------+
+| slot3 | Second image's secondary slot (internal or external flash)  | "image-3" | 4     |
++-------+-------------------------------------------------------------+-----------+-------+
+
+.. note::
+   The default upload target for firmware updates using the **mcumgr-cli** tool and the ``CONFIG_MCUBOOT_SERIAL_DIRECT_IMAGE_UPLOAD`` Kconfig option enabled is the secondary slot of the first image, denoted as slot1.
+   In this context, the default behavior without specifying a slot number corresponds to ``-n 0``.
+
+   For non-default behavior targeting the secondary slot of an image other than the first, use ``-n 2``.
+   This adjustment is due to the default slot numbering and the shift in identifiers when a non-default upload to the secondary slot is selected.
+
+For the nRF5340 DK, serial recovery differs based on the operation mode:
+
+* In the case of serial recovery:
+
+  * For updates to the network core, specify ``-n 3``.
+    This indicates that updates are mediated through the application core's RAM, a process specific to the nRF5340 DK in its default configuration.
```

Comment:
This should be moved together in "For MCUboot".

Explanation:
Serial recovery is a mode of MCUboot that allows you to "recover" board, which basically means directly load software using simple user available UART protocol, so user does not have to connect JTAG to board. This can be used to recover board or just load software without booting application first.
Normally boot process looks like this: board starts MCUboot, MCUboot searches for apps or possible update to do, then it boots application.
MCUboot is very simple and can not provide means of update like Bluetooth, etc, so we provide the optional serial recovery. When you trigger the serial recovery then MCUboot will not boot application, it will wait for commands on serial, the same type of commands normally sent to Application; because, at this point, only MCUboot is running it can write wherever it wants, with some restrictions, so it can write over application primary slot, as it is not executing it (slot0), something application can not do itself.
Also, because the mode is called recovery, it assumes that you want to fix your primary app, that is why it by default goes for slot0, rather than how app does.